### PR TITLE
Readme: update to reflect new rules + new example

### PR DESCRIPTION
**Pull request by DFoxinator**
@be-dmitry 
@bmax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ Right now it comes with the following rules (though it's easy to add your own to
 <tr><td>BetweenTimesRule</td><td>grants access if the current date/time is between two specified date/times</td></tr>
 <tr><td>StartTimeRule</td><td>grants access beginning at a specified date/time</td></tr>
```
Comment (be-dmitry) [inline README.md]: Let's specify that this compared the specified time to the current time

File: README.md [hunk 2]
```diff
@@ -15,6 +15,11 @@ Right now it comes with the following rules (though it's easy to add your own to
...
 <tr><td>EndTimeRule</td><td>grants access up until a specified end date/time</td></tr>
```
Comment (be-dmitry) [inline README.md]: Same here

**Comment by bmax** [comment]
👍

**Comment by be-dmitry** [comment]
👍

**Pull request merged**